applications/DEMApplication/tests/DEM2D_tests_files/impact2DDEM_FEM_boundary.mdpa

```diff
@@ -14,9 +14,9 @@ POISSON_RATIO 0.25
 End Properties
 
 Begin Nodes
-14 -3 1 1
-15 -3 -1 1
-16 -3 0 -1
+14 -1 1 1
```

Comment:
This file is prepared for 3D, however, the folder and the name of the case make reference to 2D

Reply:
a group of 2D particles were impacting on a RigidFace3D3N. The reference values are the same but changed it to RigidEdge2D2N for consistency